examples/scraping-zarf-agent/README.md

```diff
@@ -0,0 +1,76 @@
+import ExampleYAML from "@site/src/components/ExampleYAML";
+
+# Scrape Zarf Agent
```

Comment:
```suggestion
# Zarf Agent Metrics (Prometheus)
```
We may want to rename this to be more explicit in what we are scarping / how (similar to the `Data Injection (Kiwix)` example)